notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>I suggest to merge this chapter with next one. Those look redundant to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>done!</p>